helpers/freedom_helper.js

```diff
@@ -47,7 +47,8 @@ async function get_oauth_access_token(params) {
 }
 
 async function get_user_information({ user_id, access_token }) {
-    let url = `${base_url}${endpoints.USER_INFORMATION}${(user_id ? `/${user_id}` : '')}`;
+    let url = `${base_url}${endpoints.USER_INFORMATION}${(user_id ? `/${user_id}` : '')}?access_token=${access_token}`;
```

Comment:
Why is access_token part of URL?

Shouldn't Access token part of header?

Reply:
Yeah you are right. Moved the access_token to headers.